ChatApp/src/screens/phone_auth/otp_signin/otp_verify.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useRef, useState, useEffect } from "react";
+import { View, Text, SafeAreaView, TextInput, KeyboardAvoidingView, Platform, TouchableOpacity, Alert } from "react-native";
+import Button1 from "../../../components/buttons/button1";
+import { useNavigation } from '@react-navigation/native';
+import { useRoute } from '@react-navigation/native';
```

Comment:
Merge the imports
import { useNavigation,useRoute } from '@react-navigation/native';